src/wrapped/gnulib_qsort.c

```diff
@@ -0,0 +1,252 @@
+/* Copyright (C) 1991-2021 Free Software Foundation, Inc.
```

Comment:
I cannot accept this file. GPL is not compatible with MIT licence used in box86.

Reply:
Okay, I wrote a custom `qsort_r()`.